### DISCUSSION TITLE
[Question]: Custom Wrapper possible?

### DISCUSSION THREAD

**Discussion by georgkaser** [GetPublii/Publii#1407] [Help]
### Question

Is it possible to add custom wrapper elements to the Editor and if yes, can you tell me how to do? Like that: https://www.tiny.cloud/docs/configure/content-formatting/#exampleusingwrapper

I did'nt find something in the documentation and in the section "customElements" it seems that it's not working... Thank you :)

**Comment by candidexmedia** [comment]
It is! See my gist: https://gist.github.com/candidexmedia/367711a80274b4c005fd5983df97d20c
